packages/calico/extra/dcos-calico-confd.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=DC/OS Calico Node: A simple configuration management tool providing dynamic configurations for Calico Bird
+ConditionPathExists=/opt/mesosphere/etc/bird_enabled
+ConditionPathExists=/opt/mesosphere/etc/calico_enabled
+
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/calico-node.env
+ExecStartPre=/usr/bin/mkdir -p /etc/calico
+ExecStartPre=/usr/bin/cp /opt/mesosphere/active/calico/confd /etc/calico/confd
```

Comment:
`calico-node confd` uses `/etc/calico/confd` as the configuration path, which is unconfigurable.